bridge/bridge.go

```diff
@@ -11,8 +11,15 @@ import (
 	"strconv"
 	"strings"
 	"sync"
+	"time"
 
 	dockerapi "github.com/fsouza/go-dockerclient"
+	"github.com/rancher/go-rancher-metadata/metadata"
+)
+
+const (
+	metadataURL = "http://rancher-metadata/2015-12-19" 
```

Comment:
Hi @jnovack, Thanks for reviewing. 
No reason - that was the information I had at the time. 
I'll take another look. 
JG